latch_cli/snakemake/workflow.py

```diff
@@ -453,39 +455,46 @@ def get_fn_code(
             snakefile = Path("{snakefile_path}")
 
             lp = LatchPersistence()
+            pkg_root = Path(".")
             """,
             1,
         )
 
         code_block += reindent(
             r"""
-            pkg_root = Path(".")
 
-            exec_id_hash = hashlib.sha1()
-            token = os.environ["FLYTE_INTERNAL_EXECUTION_ID"]
-            exec_id_hash.update(token.encode("utf-8"))
-            version = exec_id_hash.hexdigest()[:16]
-
-            jit_wf_version = os.environ["FLYTE_INTERNAL_TASK_VERSION"]
-            res = execute(
-                gql.gql('''
-                query executionCreatorsByToken($token: String!) {
-                    executionCreatorByToken(token: $token) {
-                        flytedbId
-                        info {
-                            displayName
-                        }
-                        accountInfoByCreatedBy {
-                            id
+            if dry_run:
+                token = None
```

Comment:
probably still want to put the token there in case we want to download the generated entrypoint at some point

Reply:
hmm the token is retrieved from the "FLYTE_INTERNAL_EXECUTION_ID", which we don't have since this isn't running in a flyte context. We are still uploading the generated entrypoint file to `.snakemake_latch/workflows/<workflow_name>/0.0.0-dry/jit-dry-run/entrypoint.py`

I could change it to generate a new hash even for the dry run to avoid overwriting previous values, but I figured it might be simpler for the developer to just look at one place for generated code instead of having to figure out which version hash corresponds to which dry run.

Reply:
i meant the token that we store in `~/.latch/token`, tho i suppose this is taken care of already on the centromere side so nvm